setup/Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles/Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles.csproj

```diff
@@ -8,7 +8,8 @@
     <AddVsixForSigningDependsOn>SwixBuild</AddVsixForSigningDependsOn>
 
     <!-- VS Insertion -->
-    <VisualStudioInsertionComponent>Microsoft.VisualStudio.ProjectSystem.Managed</VisualStudioInsertionComponent>
+    <!-- NOTE: Having VisualStudioInsertionComponent defined is the condition for VisualStudio.SetupPackage.vsmanproj to be built (via FinalizeInsertionVsixFiles target in VisualStudio.Insertion.proj) in RepoToolset. -->
+    <VisualStudioInsertionComponent>Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles</VisualStudioInsertionComponent>
```

Comment:
I believe this to be a copy/paste bug from 2017. AFAIK, this should be the name of the insertion metadata/vsman we produce.
![image](https://user-images.githubusercontent.com/17788297/156897382-54431f01-d092-4988-8d03-00bf5793a9bf.png)

Reply:
This was an incorrect assumption. It seems we produce 3 VSIX files and 2 VSMAN files, where the `Managed.vsman` contains both the `CommonFiles` and `ProjectSystem` VSIX packages. From a build prior to these changes, you can see the VSMAN files underlined in red and the VSIX packages underlined in yellow.

![Screenshot 2022-03-10 172643](https://user-images.githubusercontent.com/17788297/157788626-c8e776e7-b21d-4241-9700-a0c419d21664.png)